test_projects/expected_state/.dockerignore

```diff
@@ -0,0 +1,23 @@
+.env
+.env.*
+.tool-versions
+*.sql
+*.md
+*.pgdump
+.gitignore
+.DS_Store
+Dockerfile
+Makefile
+README*
+docker-compose.yml
+.git/
+_docker/
+.circleci/
+_build/
+deps/
+doc/
+cover/
+priv/static/
+terraform/
+kubernetes/
```

Comment:
I believe we won't need this in the foreseeable future

Reply:
OTOH, they don't do any harm, so we might keep them. I'm fine either way.

Reply:
`kubernetes` is a bad name for this - I'm not sure what is a better name. For several Banyan Medical projects we're considering moving to Kubernetes, and in such cases it would be good to have a directory where the Helm chart would reside in.

This directory would always pretty much be the same (files generated by Helm - deployment.yaml, values.yaml, etc.).

I don't know if there's a standard practice in the Phoenix community on where the deployment configuration is supposed to reside.